src/components/Common/Button/Button.css

```diff
@@ -0,0 +1,25 @@
+.button-1 {
+  font-size: 11px;
+  background-color: black;
+  border: none;
+  color: white;
+  padding: 1rem;
+  width: 100%;
+  height: 37px;
+  border-radius: 8px;
+}
+
+.button-2 {
+  width: 100%;
+  color: #076ce0;
```

Comment:
You could use css variables for the colors so you don't need to copy the value each time you use it, also it's easier to change its value if you ever need to.